src/main/java/com/buuz135/industrial/utils/BlockUtils.java

```diff
@@ -86,7 +97,7 @@ public static boolean isBlockOreDict(World world, BlockPos pos, String ore) {
     }
 
     public static boolean isLog(World world, BlockPos pos) {
-        return (world.getBlockState(pos).getMaterial() == Material.WOOD || isBlockOreDict(world, pos, "blockSlimeCongealed"));
+        return isBlockTag(world, pos, LOGS_TAG);
```

Comment:
Instead of locally storing `LOGS_TAG` this probably should be `BlockTags.LOGS`

Reply:
Thank you for comments.
Done!